source/reference/files/ondemand-d-ymls.rst

```diff
@@ -3,64 +3,127 @@
 ondemand.d/\*.yml files
 =======================
 
-Some configurations are held within yml files in the ``/etc/ood/config/ondemand.d/`` directory.
+Most of the configurations are now held within yml files in the ``/etc/ood/config/ondemand.d/`` directory.
 Open OnDemand will read all the ``.yml`` and ``.yml.erb`` files within this directory for
 configurations.
 
 To use a different directory other than this use the ``OOD_CONFIG_D_DIRECTORY`` environment variable
 in the ``/etc/ood/config/apps/dashboard/env`` file.
 
-.. describe:: pinned_apps (Array<Object>, null)
+These properties support profile based configuration, see the :ref:`profile configuration documentation. <profiles_guide>`
 
-    An array of pinned app objects specifying what apps to pin to the dashboard.
-    See the :ref:`documentation on pinned apps <dashboard_pinned_apps>` for details
-    and examples.
+.. _profile_properties:
 
+Configuration Properties with profile support
+---------------------------------------------
+
+.. describe:: dashboard_header_img_logo: (String, null)
```

Comment:
```suggestion
.. describe:: dashboard_header_img_logo (String, null)
```

I don't think you did this. Indeed, I think I did when I started this file, but I'm just noticing it now that the other pages don't use a colon here.  Let's remove it just for consistency.

Reply:
Fixed